src/libraries/System.ComponentModel.Annotations/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,2 @@
+# Exposed publicly in the implementation to suppress linker warnings.
+TypesMustExist : Type 'System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute' does not exist in the reference but it does exist in the implementation.
```

Comment:
Should the type be public in CoreLib only, and be internal everywhere else to make this unnecessary?

It is what we do in other similar cases. For example: https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/DynamicallyAccessedMembersAttribute.cs#L28